config/locales/en/common.yml

```diff
@@ -4,6 +4,7 @@ en:
     'no': 'No'
     edit: 'Edit'
     delete: 'Delete'
+    delete_confirm: 'Are you sure to delete?'
```

Comment:
Sorry, I'm not sure I get you, simple form doesn't define these default values internally ?